extension/contentScript.js

```diff
@@ -0,0 +1,88 @@
+let messageListener = false;
+
+if (!messageListener) {
+  window.addEventListener(
+    'message',
+    (messageEvent) => {
+      if (messageEvent.source === window) {
+        chrome.runtime.sendMessage(messageEvent.data);
+      }
+    },
+    false
```

Comment:
What does the 'false' do here?

Reply:
Per MDN: A boolean value indicating that the listener should be invoked at most once after being added.  It is optional and the default value is set to false so I will remove it.